partcad/src/partcad/ai_google.py

```
@@ -93,21 +93,25 @@ def generate_google(
 
-        image_content = []
-
-        def insert_image(match):
-            filename = match.group(1)
-            image_content.append(pil_image.open(filename))
-            return "IMAGE_INSERTED_HERE"
-
-        prompt = re.sub(r"INSERT_IMAGE_HERE\(([^)]*)\)", insert_image, prompt)
-        text_content = prompt.split("IMAGE_INSERTED_HERE")
-
+        def handle_content(content):
```

Ruff (0.8.2)

Missing return type annotation for private function `handle_content`

(ANN202)